ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md

```diff
@@ -370,8 +370,8 @@ This source of truth means you have a complete overview of your entire designed
 
 Below you will find two examples of documentation automatically created by Ansible AVD:
 
-- [5 stage CLOS fabric across two Data Centers](../../molecule/eos_designs-twodc-5stage-clos/documentation/fabric/TWODC_5STAGE_CLOS-documentation.md)
-- [A single leaf inside the fabric above](../../molecule/eos_designs-twodc-5stage-clos/documentation/devices/DC1-POD1-L2LEAF1A.md)
+- [5 stage CLOS fabric across two Data Centers](https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/molecule/eos_designs-twodc-5stage-clos/documentation/fabric/TWODC_5STAGE_CLOS-documentation.md)
```

Comment:
I think we should just point these two links to the l3ls example files. Those are still part of mkdocs, and it will make it cleaner.

Reply:
The documentation files from those examples are not included. We could leave them in, we will lose a little bit of the search improvements.

Reply:
Hmm actually, its really not bad, I'll update the links and push an update.